wgpu/src/backend/web.rs

```diff
@@ -1811,8 +1807,15 @@ impl crate::Context for Context {
                     crate::LoadOp::Load => wasm_bindgen::JsValue::from(web_sys::GpuLoadOp::Load),
                 };
 
-                let mut mapped_color_attachment =
-                    web_sys::GpuRenderPassColorAttachment::new(&load_value, &ca.view.id.0);
+                let mut mapped_color_attachment = web_sys::GpuRenderPassColorAttachment::new(
+                    &load_value,
+                    if ca.ops.store {
```

Comment:
We can use `map_store_op` for this

Reply:
I've added the call to `map_store_op`, good catch, didn't see that it existed.